src/picker-popup.html

```diff
@@ -51,23 +51,23 @@
         <label class="col col-20">
             <div class="item item-input">
                 <div>
-                    <input type="text" ng-model="bind.hour" pattern="0?([01]?[0-9]|2[0-3])" ng-change="change('hour')" ng-blur="changed()" required>
+                    <input type="{{timeFieldType}}" ng-model="bind.hour" pattern="0?([01]?[0-9]|2[0-3])" ng-change="change('hour')" ng-blur="changed()" required>
```

Comment:
Please use `ng-attr-type=` instead of `type=`, there were [some problems](http://stackoverflow.com/questions/23941699/setting-input-type-file-using-angularjs-expression-not-working-in-chrome) in older browsers with it.